templates/quickstart-jira-dc-with-vpc.template.yaml

```
@@ -0,0 +1,728 @@
+        - QSS3Region: !If
+            - GovCloudCondition
+            - s3-us-gov-west-1
+            - s3
+      Parameters:
+        AssociatePublicIpAddress: !Ref 'AssociatePublicIpAddress'
+        CatalinaOpts: !Ref 'CatalinaOpts'
+        CidrBlock: !Ref 'CidrBlock'
+        ClusterNodeInstanceType: !Ref 'ClusterNodeInstanceType'
+        ClusterNodeMax: !Ref 'ClusterNodeMax'
+        ClusterNodeMin: !Ref 'ClusterNodeMin'
+        ClusterNodeVolumeSize: !Ref 'ClusterNodeVolumeSize'
+        CustomDnsName: !Ref 'CustomDnsName'
+        DBInstanceClass: !Ref 'DBInstanceClass'
+        DBIops: !Ref 'DBIops'
+        DBMasterUserPassword: !Ref 'DBMasterUserPassword'
+        DBMaxIdle: !Ref 'DBMaxIdle'
+        DBMaxWaitMillis: !Ref 'DBMaxWaitMillis'
+        DBMinEvictableIdleTimeMillis: !Ref 'DBMinEvictableIdleTimeMillis'
+        DBMinIdle: !Ref 'DBMinIdle'
+        DBMultiAZ: !Ref 'DBMultiAZ'
+        DBPassword: !Ref 'DBPassword'
+        DBPoolMaxSize: !Ref 'DBPoolMaxSize'
+        DBPoolMinSize: !Ref 'DBPoolMinSize'
+        DBRemoveAbandoned: !Ref 'DBRemoveAbandoned'
+        DBRemoveAbandonedTimeout: !Ref 'DBRemoveAbandonedTimeout'
+        DBStorage: !Ref 'DBStorage'
+        DBStorageEncrypted: !Ref 'DBStorageEncrypted'
+        DBStorageType: !Ref 'DBStorageType'
+        DBTestOnBorrow: !Ref 'DBTestOnBorrow'
+        DBTestWhileIdle: !Ref 'DBTestWhileIdle'
+        DBTimeBetweenEvictionRunsMillis: !Ref 'DBTimeBetweenEvictionRunsMillis'
+        DeployEnvironment: !Ref 'DeployEnvironment'
+        HostedZone: !Ref 'HostedZone'
+        JiraDownloadUrl: !Ref 'JiraDownloadUrl'
+        JiraProduct: !Ref 'JiraProduct'
+        JiraVersion: !Ref 'JiraVersion'
+        JvmHeapOverride: !Ref 'JvmHeapOverride'
+        KeyName: !Ref 'KeyName'
+        LocalAnsibleGitRepo: !Ref 'LocalAnsibleGitRepo'
+        LocalAnsibleGitSshKeyName: !Ref 'LocalAnsibleGitSshKeyName'
+        MailEnabled: !Ref 'MailEnabled'
+        SubDomainName: !Ref 'SubDomainName'
+        SSLCertificateARN: !Ref 'SSLCertificateARN'
+        StartCollectd: !Ref 'StartCollectd'
+        TomcatAcceptCount: !Ref 'TomcatAcceptCount'
+        TomcatConnectionTimeout: !Ref 'TomcatConnectionTimeout'
+        TomcatContextPath: !Ref 'TomcatContextPath'
+        TomcatDefaultConnectorPort: !Ref 'TomcatDefaultConnectorPort'
+        TomcatEnableLookups: !Ref 'TomcatEnableLookups'
+        TomcatMaxThreads: !Ref 'TomcatMaxThreads'
+        TomcatMinSpareThreads: !Ref 'TomcatMinSpareThreads'
+        TomcatProtocol: !Ref 'TomcatProtocol'
+        TomcatRedirectPort: !Ref 'TomcatRedirectPort'
+        TomcatScheme: !Ref 'TomcatScheme'
+
+Outputs:
+  ServiceURL:
+    Description: The URL to access this Atlassian service
+    Value: !GetAtt 'JiraDCStack.Outputs.ServiceURL'
+  LoadBalancerURL:
+    Description: The Load Balancer URL
+    Value: !GetAtt 'JiraDCStack.Outputs.LoadBalancerURL'
+  SGname:
```

When we are creating the VPC stack as part of the template I found it very valuable for testing and further investigation to add `BastionPubIp` from the Bastion stack. This is now enabled with this change being merged -https://github.com/aws-quickstart/quickstart-atlassian-services/blob/master/templates/quickstart-vpc-for-atlassian-services.yaml#L226 (Outputs doesn't support referencing value from the second level).
Something like
```suggestion
  BastionIP:
    Description: Bastion node IP (use as a jumpbox to connect to the nodes)
    Value: !GetAtt 'VPCStack.Outputs. BastionPubIp'
```